src/components/menu/menu.config.js

```diff
@@ -1,6 +1,7 @@
 export default {
   light: {
     background: "var(--colorsComponentsMenuSpringStandard500)",
+    borderRadius: "var(--borderRadius050)",
```

Comment:
this can just be defined in the styles as it's not dependant on the menuType prop value